src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,141 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final int MULTIPLICATION_INDEX = 2;
+    private int size;
+    private Node<K, V>[] table = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int threshold = (int) (table.length * DEFAULT_LOAD_FACTOR);
+
+    class Node<K, V> {
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(K key, V value, Node<K,V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+
+        public final K getKey() {
+            return key;
+        }
+
+        public final V getValue() {
+            return value;
+        }
```

Comment:
we can access key and value from myHashMap without getters because its inner class and its private fields are available